src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -86,6 +102,59 @@ public void handlesSocket() throws IOException {
     }
 
     /**
+     * BkBasic can handle HTTP persistent connections.
+     * @throws Exception If some problem inside
+     * @checkstyle ExecutableStatementCountCheck (2 lines)
+     */
+    @Test
+    @SuppressWarnings("PMD.DoNotUseThreads")
+    public void handlesPersistentConnection() throws Exception {
+        final int port = 8080;
```

Comment:
@yegor256 I can detect a free port but it will increase complexity of BkBasicTest. According to your comment ( https://github.com/yegor256/takes/pull/337#issuecomment-112151309 ) I need to run the server for testing persistent connection. Do you have any suggestion ? Should i detect a free port ?